doc/editorconfig.txt

```diff
@@ -84,6 +84,15 @@ If you choose "external_command" mode, you must also set
 Changes to "g:EditorConfig_core_mode" will not take effect until Vim
 is restarted.
 
+                                            *b:EditorConfig_disable*
+This is a buffer-local variable that disables the EditorConfig plugin for a
+single buffer.
+
+Example: Disable EditorConfig when drafting a git commit message
```

Comment:
I like this example, but it is an example of disabling for _all_ git commit messages.  Would you please add another example above this one that just shows `let b:EditorConfig_disable=1` (the single-buffer case)?